jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -327,7 +327,7 @@ def _roman_2d_asdf_to_glue_data(file_obj, data_label, ext=None):
         component = Component.autotyped(np.array(getattr(file_obj, ext)), units=bunit)
         data.add_component(component=component, label=comp_label)
         meta = getattr(file_obj, 'meta')
-        data.coords = getattr(meta, 'wcs')
+        data.coords = getattr(meta, 'wcs', None)
```

Comment:
Currently the dev version of `roman_datamodels` has a method for generating WFI-like data products, but it doesn't generate artificial WCS yet. I've created an issue over there for requesting this: https://github.com/spacetelescope/roman_datamodels/issues/134.